src/app/components/address-identicon/address-identicon.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container" [style.width]="size" [style.height]="size">
+    <img
+        class="container__identicon"
+        [src]="identicon(address)"
+        alt="Channel partner identicon"
```

Comment:
This would "limit" (not actually) the component to be used for channel partners only. So I think this is incorrect, isn't it? Couldn't such identicon also be used for the own account or for transfer targets or...?

Reply:
Oh yes, you are right. It is indeed used in other places than a channel.